optuna/cli.py

```diff
@@ -147,7 +147,6 @@ def take_action(self, parsed_args):
         elif parsed_args.study:
             message = "The use of `--study` is deprecated. Please use `--study-name` instead."
             warnings.warn(message, DeprecationWarning)
```

Comment:
Could you replace `DeprecationWarning` with `FutureWarning` to show it on the screen?
(c.f., https://github.com/optuna/optuna/pull/1428)
```suggestion
            warnings.warn(message, FutureWarning)
```